build.gradle

```diff
@@ -1670,6 +1714,10 @@ project(':server-common') {
   checkstyle {
     configProperties = checkstyleConfigProperties("import-control-server-common.xml")
   }
+
+  javadoc {
+    enabled = false
```

Comment:
We don't have public classes in `server-common`, so make it explicit that javadoc should not be published.